aio_pika/tools.py

```diff
@@ -166,3 +166,4 @@ def __call__(self, *args, **kwargs):
                     cb(self.__sender(), *args, **kwargs)
                 except Exception:
                     log.exception("Callback error")
+                    raise
```

Comment:
It breaks the callback collection loop, the other callbacks will skip after a bad one. It's breaking change.

Reply:
Canceled these changes. Thanks for the review!